front/cypress/e2e/form_builder/elements/single_choice_field.cy.ts

```diff
@@ -62,6 +62,38 @@ describe('Form builder single choice field', () => {
     cy.get('#e2e-single-select-control').should('exist');
   });
 
+  it('allows submitting when there is an other option but it is not filled out', () => {
```

Comment:
Minor: I would add not selected and not filled out
```suggestion
  it('allows submitting when there is an other option that is not selected and is not filled out', () => {
```